registry/storage/registry.go

```diff
@@ -132,6 +142,18 @@ func (reg *registry) Repository(ctx context.Context, name string) (distribution.
 	}, nil
 }
 
+// RegistryBlobKeeper returns an instance of BlobKeeper for given registry object.
+func RegistryBlobKeeper(ns distribution.Namespace) (distribution.BlobKeeper, error) {
+	reg, ok := ns.(*registry)
+	if !ok {
+		return nil, fmt.Errorf("Cannot instantiate BlobKeeper with given namespace object (%T)", ns)
```

Comment:
error messages in `errors.New()` or `fmt.Errorf()` should start with a lowercase letter